tests/cases/fourslash/codeFixInferFromUsagePropertyAccess.ts

```diff
@@ -12,4 +12,4 @@
 ////    return x.y.z
 ////}
 
-verify.rangeAfterCodeFix("a: { b: { c: void; }; }, m: { n: () => number; }, x: { y: { z: number[]; }; }", /*includeWhiteSpace*/ undefined, /*errorCode*/ undefined, /*index*/0);
+verify.rangeAfterCodeFix("a: { b: { c: any; }; }, m: { n: () => number; }, x: { y: { z: number[]; }; }", /*includeWhiteSpace*/ undefined, /*errorCode*/ undefined, /*index*/0);

```

Comment:
This looks like an unintended positive bug fix, right? Other places (e.g. `codeFixInferFromCallInAssignment`) infer `any` for variable types that would otherwise be describable as `unknown`.

Reply:
I'm not sure what you mean by "unintended positive". `void` was wrong and `any` is right as far as I can tell.

I think, in most positions, inferring `any` vs `unknown` is a matter of preference -- how strict do you want the result to be? Right now I assume that people want pretty loose results, since I expect people will use this after converting from .js to .ts, and before trying to get rid of any explicit `any`.

We could discuss adding another fix to the codefix, one that defaults to `unknown` when possible. I think it would be a decent idea, but hard to explain in the space of a codefix title.

Reply:
Oh yes I think we're on the same page - I meant that the change from `void` to `any` is positive, and I didn't initially intend it as part of this PR's changes. 👍